docker-files-for-Gitlab-CI-rust/cross/snapcraft/Dockerfile

```diff
@@ -7,15 +7,23 @@ MAINTAINER Parity Technologies <devops@parity.io>
 RUN apt-get update && \
         apt-get dist-upgrade --yes && \
 	apt-get install --yes --no-install-recommends \
-        snapcraft git snapd ca-certificates file python zip curl expect nodejs cmake wget \
-	&& \
+        snapcraft \
+        git \
+        snapd \
+        ca-certificates \
+        file \
+        python \
+        zip \
+        curl \
+        expect \
+        gettext-base \
+        nodejs \
+        cmake \
+        wget && \
 	wget http://security.ubuntu.com/ubuntu/pool/universe/s/snapcraft/snapcraft_2.43.1_all.deb &&\
 	dpkg -i snapcraft_2.43.1_all.deb &&\
```

Comment:
TODO - Until this version appears in the stable

Reply:
Yes, but we can leave it like this for now, can't we?